extensions/wikia/AdEngine/js/video/player/playerTracker.js

```diff
@@ -3,23 +3,23 @@ define('ext.wikia.adEngine.video.player.playerTracker', [
 	'ext.wikia.adEngine.adContext',
 	'ext.wikia.adEngine.adLogicPageParams',
 	'ext.wikia.adEngine.adTracker',
-	'ext.wikia.adEngine.lookup.prebid.bidHelper',
 	'ext.wikia.adEngine.slot.slotTargeting',
 	'wikia.browserDetect',
 	'wikia.geo',
 	'wikia.log',
 	'wikia.window',
+	require.optional('ext.wikia.adEngine.lookup.prebid.bidHelper'),
 	require.optional('ext.wikia.adEngine.video.player.porvata.floater')
 ], function (
 	adContext,
 	pageLevel,
 	adTracker,
-	bidHelper,
 	slotTargeting,
 	browserDetect,
 	geo,
 	log,
 	win,
+	bidHelper,
```

Comment:
What's the reason for moving it here?A race condition when loading {{allinone=0}}?